packages/walletconnect-core/.babelrc

```diff
@@ -1,4 +1,14 @@
 {
-  "presets": ["@babel/preset-env"],
-  "plugins": ["add-module-exports"]
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "modules": false,
+        "targets": {
+          "browsers": ["last 2 versions"],
```

Comment:
"browsers": [
  ">0.25%",
  "not ie 11",
  "not op_mini all"
]

Reply:
Source: https://jamie.build/last-2-versions

Reply:
The coverage for "last 2 versions" is 86.71% and this configuration is 85.86%, but the difference in bundle size is significant since it doesn't support deprecated browsers.